cucumber/authenticators_gce/features/step_definitions/authn_gce_steps.rb

```diff
@@ -14,19 +15,44 @@
     raise "Incorrect annotation name: '#{annotation_name}', expected: service-account-id|service-account-email|project-id|instance-name"
   end
 
-  set_annotation_to_resource("#{annotation_name}", annotation_value)
+  set_annotation_to_resource("authn-gce/#{annotation_name}", annotation_value)
 end
 
-Given(/^I obtain a GCE identity token in "([^"]*)" format with audience claim value: "([^"]*)"$/) do | audience, format |
+# Sets all GCE annotations
+Given(/^I set GCE annotations to host "([^"]*)"$/) do | hostname |
```

Comment:
Do we want it will verify all annotations ? i prefer it will be one by one, otherwise its not clear by reading the feature file only.

Reply:
There's also a method to set explicit.

Reply:
than maybe change the name to obtain all CGE annotations to host

Reply:
You probably meant  `set all...` NP.